modules/cassandra/src/test/java/org/testcontainers/containers/CassandraContainerTest.java

```diff
@@ -0,0 +1,125 @@
+package org.testcontainers.containers;
+
+import com.datastax.driver.core.Cluster;
+import com.datastax.driver.core.ResultSet;
+import com.datastax.driver.core.Row;
+import com.datastax.driver.core.Session;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.CassandraQueryWaitStrategy;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author Eugeny Karpov
+ */
+public class CassandraContainerTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(CassandraContainerTest.class);
+    private static final String TEST_CLUSTER_NAME_IN_CONF = "Test Cluster Integration Test";
+
+    @Test
+    public void testSimple() throws Exception {
+        CassandraContainer cassandraContainer = (CassandraContainer) new CassandraContainer()
+                .withLogConsumer(new Slf4jLogConsumer(logger));
+        performWithContainer(cassandraContainer, () -> {
```

Comment:
why not just (as in the other tests):
```
try(CassandraContainer<?> container = new CassandraContainer<>()) {
  container.start();
  // Do something with container
}
```
?

Reply:
Before writing tests I saw SimpleMySQLTest. Btw personally I don't like try with resources if class uses builder initialization - it becomes a little clumsy and hard to read. That's why I used lambdas.
Ok, lambdas replaced by proposed try-with-resources style.